applications/openshift/classification-banner/rule.yml

```diff
@@ -0,0 +1,25 @@
+prodtype: ocp4
+
+title: Security Banner must be configured on OpenShift console
+
+description: test
+
+rationale: TBD
+
+identifiers: {}
+
```

Comment:
given that we know what controls this is addressing, let's add the `references` key and state the nist control.